scipy/ndimage/tests/test_fourier.py

```diff
@@ -1,137 +1,181 @@
+import math
 import numpy as np
-from numpy import fft
-from numpy.testing import (assert_almost_equal, assert_array_almost_equal,
-                           assert_equal)
+
+from scipy._lib._array_api import (
+    xp_assert_equal,
+    assert_array_almost_equal,
+    assert_almost_equal,
+)
+from scipy._lib._array_api import is_cupy
 
 import pytest
 
 from scipy import ndimage
 
+from scipy.conftest import array_api_compatible
+skip_xp_backends = pytest.mark.skip_xp_backends
+pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_xp_backends")]
```

Comment:
looks like the skip infrastructure is unused here? Not sure why the linter hasn't picked this up.

Reply:
Keep linter improvements for a follow-up then?